deploy/role.yaml

```
@@ -32,8 +32,7 @@ rules:
   resources:
   - servicemonitors
   verbs:
-  - get
-  - create
+  - '*'
```

Per https://github.com/operator-framework/operator-sdk/issues/671, can we swap out the * for:
```suggestion
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
```